src/policy.c

```diff
@@ -831,6 +831,33 @@ move_installed_to_front(Pool *pool, Queue *plist)
     }
 }
 
+
+/*
+ * prune_to_considered
+ *
+ * Keep only considered (pool->considered) solvable in plist. If pool->considered is NULL it keeps
+ * all solvable in plist.
+ */
+static void
+prune_to_considered(Pool *pool, Queue *plist)
+{
+  if (plist->count < 2)		/* no need to prune for a single entry */
```

Comment:
I don't think this belongs to this function.

Reply:
It probably makes sense, since that prevents it from filtering to zero on accident.

Reply:
Thanks for pointing here, because for correct functionality the condition should be ``if (plist->count == 0)``. It is really required to remove all excluded solvables for update job even if it ends up in empty plist. The install job will report (will only not mention the best rule)
```
Problem: conflicting requests
  - package TestA-1.0.0-1.noarch is excluded
  - package TestA-1.0.0-2.noarch is excluded
```